### PR TITLE
added new page and link in navigation

### PR DESCRIPTION
**Pull request by florianblaschke**

### CODE REVIEW

File: lib/functions.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+      roast.score
+        .map(({ rating }) => rating)
+        .reduce((acc, curr) => acc + curr, 0) / roast.score.length;
+    return { ...roast, reducedScore: reducedScore };
+  });
+}
+export const shuffle = (array) => {
+  for (let i = array.length - 1; i > 0; i--) {
+    const j = Math.floor(Math.random() * (i + 1));
+    [array[i], array[j]] = [array[j], array[i]];
```
Comment (Kheirah) [inline lib/functions.js:14]: Destructuring to swap elements, nice!

File: lib/functions.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  }
+  return array;
+};
+
+export function sortedForRating(data) {
```
Comment (Kheirah) [inline lib/functions.js]: If you like you can add the sorting direction to the name of the function, something like `sortedForRatingDesc` which informs the developer about the fact that the array will be sorted in descending order.

**Review by Kheirah — APPROVED** [review]
The search logic can be slimmed down for sure, but let's leave it for now. Your solutions works and that's important! 
I'll have to look into again to give you more details for how to refactor it.

File: lib/functions.js (hunk 1 above)
Comment (florianblaschke) [reply lib/functions.js:14]: Fisher-Yates Shuffle! Not my own creation but I really liked the solution ;)

**Comment by florianblaschke** [comment]
I am still open to suggestions though…I was struggling to find a better way to implement the queryLinks if the user get redirected from the main page.

**Pull request merged**